iocore/hostdb/test_RefCountCache.cc

```diff
@@ -135,8 +135,6 @@ testRefcounting()
   cache->put(1, to_delete);
   ret |= to_delete->refcount() != 1;
   cache->erase(1);
-  ret |= to_delete->refcount() != 0;
```

Comment:
Since the memory is actually getting freed we should figure out a way to test that it was in fact deleted. An alternative is maybe to have it update a global map of key -> bool for things that where deleted? then we could simply do a get and ensure its not in the map. (something to replace the test instead of dropping it. Or maybe a global atomic for number of active ExampleStruct objects